subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/api/ConfigurationRoleUsageIntegrationTest.groovy

```diff
@@ -673,13 +673,13 @@ class ConfigurationRoleUsageIntegrationTest extends AbstractIntegrationSpec impl
         "using create to make an implicitly LEGACY configuration"   | "create('fooElements')"                                                                                       | "--warn"      || null
         "using consumable to make a configuration"                  | "consumable('fooElements')"                                                                                   | "--warn"      || null
         "using resolvable to make a configuration"                  | "resolvable('fooElements')"                                                                                   | "--warn"      || null
-        "using resolvable_bucket to make a configuration"           | "createWithRole('fooElements', ConfigurationRoles.INTENDED_RESOLVABLE_BUCKET)"                                | "--warn"      || null
-        "using consumable_bucket to make a configuration"           | "createWithRole('fooElements', ConfigurationRoles.INTENDED_CONSUMABLE_BUCKET)"                                | "--warn"      || null
+        "using resolvable_bucket to make a configuration"           | "createWithRole('fooElements', ConfigurationRoles.RESOLVABLE_BUCKET)"                                | "--warn"      || null
+        "using resolvable_bucket to make a configuration"           | "createWithRole('fooElements', ConfigurationRoles.CONSUMABLE_BUCKET)"                                | "--warn"      || null
```

Comment:
Did you mean to switch this to `resolvable_bucket` here and below? 

Also the table seems to be out of alignment now